src/core/pagerankGraph.test.js

```diff
@@ -63,6 +69,50 @@ describe("core/pagerankGraph", () => {
     });
   });
 
+  describe("node prefix filter matches graph filter", () => {
```

Comment:
I would slightly restructure this describe block. I think the test that verifies that a prefix is required when options are passed should be outside of the describe, then the describe block should contain just the graph<>pagerankGraph consistency checks.

Reply:
Good call!